src/definitions/modules/calendar.js

```diff
@@ -655,6 +655,11 @@ $.fn.calendar = function(parameters) {
             updateInput = updateInput !== false;
             fireChange = fireChange !== false;
             date = module.helper.sanitiseDate(date);
+            
+            if (!date) {
+                console.error('Invalid date value for calendar')
```

Comment:
You may also move the error text into the error text object (where already `error.popup` and `error.method` exist)

```javascript
module.error(error.dateValue);

...

error: {
    popup: 'UI Popup, a required component is not included in this page',
    method: 'The method you called is not defined.',
    dateValue: 'Invalid date value for calendar'
}
```